js/src/components/google-combo-account-card/connect-ads/connect-cta.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppButton from '.~/components/app-button';
+import useEventPropertiesFilter from '.~/hooks/useEventPropertiesFilter';
+import { FILTER_ONBOARDING } from '.~/utils/tracks';
+
+/**
+ * Clicking on the button to connect an existing Google Ads account.
+ *
+ * @event gla_ads_account_connect_button_click
+ * @property {number} id The account ID to be connected.
+ * @property {string} [context] Indicates the place where the button is located.
+ * @property {string} [step] Indicates the step in the onboarding process.
+ */
+
+/**
+ * Connect CTA component.
+ *
+ * @param {Object} props Props.
+ * @param {Function} props.handleConnectClick Callback to handle the connect click.
+ * @param {string} props.value Connected account ID.
+ * @fires gla_ads_account_connect_button_click when "Connect" button is clicked.
+ * @return {JSX.Element} Connect CTA component.
+ */
+const ConnectCTA = ( { handleConnectClick, value } ) => {
```

Comment:
@ankitrox Let's update `js/src/components/google-ads-account-card/connect-ads/index.js` to use `ConnectCTA`. I think it's better we have `ConnectCTA` in the following folder: `js/src/components/google-ads-account-card/connect-ads/`